apps/functions/applications-migration/common/migrators/general-s51-advice-migration.js

```diff
@@ -0,0 +1,53 @@
+import { QueryTypes } from 'sequelize';
+import { SynapseDB } from '../synapse-db.js';
+import { migrateS51AdviceForCase } from './s51-advice-migration.js';
+import { ODW_GENERAL_CASE_REF } from '../constants.js';
+
+/**
+ * @param {import('@azure/functions').Logger} logger
+ * @param {number} offset
+ */
+export const migrateGeneralS51Advice = async (logger, offset = 0) => {
+	const countQuery = `
+		SELECT COUNT(*) AS generalS51AdviceCount
+		FROM [odw_curated_db].[dbo].[nsip_s51_advice]
+		WHERE caseReference = '${ODW_GENERAL_CASE_REF}';
+	`;
+	const [{ generalS51AdviceCount }] = await SynapseDB.query(countQuery, {
+		type: QueryTypes.SELECT
+	});
+	logger.info(`Number of General S51 Advice found in odw_curated_db: ${generalS51AdviceCount}`);
+
+	const batchSize = 10;
```

Comment:
We can probably increase the batch size, although I'm not sure what the upper limit should be. Did you notice any issues when trying to process all the records in one batch?

Reply:
Also is it worth setting the batch size in some top-level config?

Reply:
yeah, I agree, 10 is definitely too low. I'll try it locally with a large batch size to see if it causes issues. I put the batching in preemptively really so the api wouldn't have to deal with ~800 advices at once, but it might be alright

Reply:
I think the main bottleneck with the API is the request body size. @HadrianHughesKC did we do some work to increase that a few months ago?

Reply:
Yeah @JamesP-KC that rings a bell. I forget how far we got with it though. IIRC the default max request body is 30mb.

Reply:
Tested it locally and it does seem like it's able to do them all in one go (bear in mind that this will be ran separately to the typical case migration). Although, a low batchsize does allow you to restart the process at a certain point if something goes wrong with the migration of an advice. Not sure how useful this will be in practice though 🤔

Reply:
upped the batchsize to 100 for now